webapp/sources/utils/src/test/scala/com/normation/ZioCommonsTest.scala

```diff
@@ -162,6 +162,18 @@ object TestImplicits {
   }
 }
 
+object SimpleEvalTest {
+
+  val hello = IOResult.effect(println("plop"))
+
```

Comment:
A reminder that effect are executed at each call of `runNow`. Really really. Even if it's a val.